### PR TITLE
Few edits in README file

### PR DESCRIPTION
**Pull request by alekhyal**
Edits include: 
1. TOC add-ons
2. Added "Create new Stackdriver Account" under Deployment - As this is necessary before we run 'make create'
3. Added few steps on how to "Configure gcloud settings" - To configure project, region, zone.
4. Few edits in "Validate" - to run 'make validate' - this is configured in Make file and a script validate.sh is present but never used.

### CODE REVIEW

**Comment by chrislovecnm** [comment]
@alekhyal you do not have git setup correctly.  Please follow the instructions in our onboarding guide and amend your commit.

**Comment by chrislovecnm** [comment]
https://github.com/GoogleCloudPlatform/gke-monitoring-tutorial/commit/5727e9e7cded06f6dff5935885941dbd16f2cf87.patch

Shows how your email address is not in the commit.  It needs to be for the CLA.

File: README.md [hunk 1]
```diff
@@ -81,8 +82,51 @@ The Terraform configuration will execute against your GCP environment and create
 $ gcloud auth application-default login
 ```
 
+### Configure gcloud settings
+
+Run `gcloud config list` and make sure that `compute/zone`, `compute/region` and `core/project` are populated with values that work for you. You can choose a [region and zone near you](https://cloud.google.com/compute/docs/regions-zones/). You can set their values with the following commands:
```
[the remark this answers is not in the document]
Comment (alekhyal) [reply README.md]: Removed these changes in the latest commit.

**Comment by geojaz** [comment]
/approve
thanks :)

**Pull request merged**